src/runtime/virtcontainers/factory/template/template.go

```diff
@@ -167,6 +175,10 @@ func (t *template) createFromTemplateVM(ctx context.Context, c vc.VMConfig) (*vc
 	config.HypervisorConfig.VMStorePath = c.HypervisorConfig.VMStorePath
 	config.HypervisorConfig.RunStorePath = c.HypervisorConfig.RunStorePath
 
+	if t.config.HypervisorType == vc.ClhHypervisor {
+		config.HypervisorConfig.DevicesStatePath = t.statePath + "/state.json"
```

Comment:
I'd also create a `const` for `state.json` and maybe a simple function to actually create this path, again to reduce duplication and potentially avoid future issues.

Reply:
Thanks for your review, already change to use const.